CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# 109.3.0
+
+This minor release includes API improvements to ProgressView, SnackBars and TextFields, as well as tests and bug fixes for ActionSheet, AppBar, BottomAppBar, BottomSheet, Collections, Dialogs, FlexibleHeader and Tabs.
+
+## New features
+
+ProgressView supports a gradient progress bar:
+
+```objc
+self.progressView.progressTintColors = @[
+    (id)UIColor.greenColor.CGColor, (id)UIColor.blueColor.CGColor, (id)UIColor.redColor.CGColor
+  ];
```

Comment:
```objc
self.progressView.progressTintColors = @[
  (id)UIColor.greenColor.CGColor, (id)UIColor.blueColor.CGColor, (id)UIColor.redColor.CGColor
];
```

Reply:
@jverkoey to be clear, this is a suggested change fixing indentation?

Reply:
Yep!